src/qt/forms/importkeysdialog.ui

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>ImportKeysDialog</class>
+ <widget class="QDialog" name="ImportKeysDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>560</width>
+    <height>440</height>
```

Comment:
Height is a bit high at the moment.

Reply:
I changed it to 200, because that's the smallest I could get it to go locally.